app/graphql/types/query_type.rb

```diff
@@ -11,19 +11,19 @@ class QueryType < Types::BaseObject
     def user(id:)
       User.find(id)
     end
-        
+
     field :competitions, [Types::CompetitionType], null: false
-    
+
     def competitions
       Competition.all
     end
 
     field :competition, Types::CompetitionType, null: false do
-      argument :id, ID, required: true
+      argument :month, Int, required: true
     end
```

Comment:
Dope, looks good; we might also want to add an argument for year, but this'll definitely do the thing for now

Reply:
Good call. I'll open up an issue for this